commands/update.bee.inc

```diff
@@ -101,3 +112,63 @@ function update_db_bee_callback() {
     bee_message(bt('All pending updates applied.'), 'success');
   }
 }
+
+/**
+ * Command callback: Show, and apply, module and theme updates.
+ *
+ * @todo  Add core update.
+ * @todo  Consider a way to download first, check the download has worked before delete.
+ *
+ */
+function update_bee_callback() {
+  global $_bee_backdrop_root;
+  require_once $_bee_backdrop_root . '/core/includes/file.inc';
+  $themes = list_themes();
```

Comment:
```suggestion
```
One more tweak before merging this into the branch.  I couldn't find any later mention of this variable. I tested the function without, even when a theme needs an update and it worked fine without.

Reply:
Should we remove the other references to 'themes' in the module too?

Reply:
I cannot remember where it came from now.
I will action your update